### PR TITLE
SQL: remove beta marker from documentation

### PR DESCRIPTION
**Pull request by costin**

### CODE REVIEW

**Comment by costin** [comment]
@elasticmachine run elasticsearch-ci/docbldesx

**Comment by costin** [comment]
@elasticmachine run elasticsearch-ci/docbldesx

File: docs/reference/sql/functions/date-time.asciidoc [hunk 1]
```diff
@@ -91,8 +89,6 @@ include-tagged::{sql-specs}/docs.csv-spec[dtIntervalMul]
 
 ==== Functions
 
-beta[]
```
Comment (bmorelli25) [inline docs/reference/sql/functions/date-time.asciidoc]: This is the problem. You have two headings of the same level with no content between them. Something needs to go in this space here. When I put a comment here: `//comment`, the docs built for me.

**Review by bmorelli25 — COMMENTED** [review]
Ahh, this is a fun one. When I built these changes locally, I got a long list of gibberish in my error. The important part is this line: `/Users/brandonmorelli/Documents/GitHub/elasticsearch/html_docs/index.xml:101739: element section: validity error`

Taking a look at line `101739` in `index.xml`, I saw the error was occurring on a line with the word `Functions` in it. Above that line was the term "Basic arithmetic operators". Searched for that term in the documentation and it led me to the `date-time.asciidoc` file. Sure enough, we can scroll to the `Functions` section and see the problem. 

Here's what this looked like before:
```
==== Functions

beta[]

[[sql-functions-current-date]]
==== `CURRENT_DATE/CURDATE`
```
And now it looks like:
```
==== Functions

[[sql-functions-current-date]]
==== `CURRENT_DATE/CURDATE`
```

It's a subtle problem, but you can't have an empty space between headers of the same level. You need to put something between `==== Functions` and `==== CURRENT_DATE/CURDATE`. Once you do that, the docs built just fine for me!

File: docs/reference/sql/endpoints/odbc/configuration.asciidoc [hunk 2]
```diff
@@ -156,7 +154,7 @@ the server that the certificate was meant for.
 * Enabled. Certificate identity chain validated.
 +
 This setting is equivalent to the previous one, with one additional check against certificate's revocation. This offers the strongest
-security option and is the recommended setting for production deployments.
+security option and is the recommended setting for odbcion deployments.
```
Comment (astefan) [inline docs/reference/sql/endpoints/odbc/configuration.asciidoc]: A small replacement failure.

File: docs/reference/sql/functions/date-time.asciidoc (hunk 1 above)
Comment (costin) [reply docs/reference/sql/functions/date-time.asciidoc]: Cheers!

**Pull request merged**